modules/server/src/main/assets/css/partials/_main.scss

```diff
@@ -126,6 +126,71 @@ main {
       text-align: center;
       margin-bottom: 20px;
     }
+    .ecosystem-heading {
+      width: 100%;
+      text-align: center;
+      font-weight: bold;
+      color: white;
+      margin: 5px;
+    }
+    .ecosystem-badges {
+      display: flex;
```

Comment:
I would rather use Bootstrap grid system than flexbox. Bootstrap is built internally using flexbox, but it is easier to work with and maintain. Especially in this case where we have a fixed number of boxes on each row.